src/tools/examples/avtexamples/CMakeLists.txt

```diff
@@ -71,13 +71,15 @@ ${VTK_EXTRA_LIBS}
 ${OPENGL_LIBRARIES}
 ${CMAKE_THREAD_LIBS}
 ${ZLIB_LIBRARY}
+${DL_LIB}
 )
 
 if(HAVE_OSMESA)
     target_link_libraries(avtFilterExample ${OSMESA_LIBRARIES})
 elseif(HAVE_EGL)
     target_link_libraries(avtFilterExample ${EGL_LIBRARIES})
 endif()
+target_link_libraries(avtFilterExample ${DL_LIB})
```

Comment:
Why does DL_LIB need to be listed twice? (here and elsewhere?)
Will placing it higher up in the list of target_link_libraries remove the need for it to be listed twice?

Reply:
Fixed.